modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCenter.java

```diff
@@ -117,9 +117,6 @@ public String getCredentialOpenApiUrl(Csp csp, CredentialType type) {
      */
     public List<AbstractCredentialInfo> listCredentials(Csp csp, CredentialType type,
```

Comment:
can you please confirm if the credential cache really works correctly when the userID is null?

Reply:
I've looked at the code and I think it works fine when userId is null
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/7a848c7c-d44b-4b76-bee2-0aa8aeb3a660)
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/cc4f5f56-4e0d-4b54-8b0e-50a70b6e10c0)
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/ffb26102-f368-476c-a712-211d95bfefa3)